src/app.rs

```diff
@@ -442,10 +442,12 @@ pub fn build_app() -> App<'static, 'static> {
                 .long_help(
                     "Filter results based on the file modification time. The argument can be provided \
                      as a specific point in time (YYYY-MM-DD HH:MM:SS) or as a duration (10h, 1d, 35min). \
-                     '--change-newer-than' can be used as an alias.\n\
+                     By default, for point in time if no time is specified (HH:MM:SS) it defaults to 00:00:00. \
```

Comment:
I don't really understand that sentence. Maybe simply "If the time is not specified, it defaults to 00:00:00."

Reply:
Yeah I like that better.